### DISCUSSION TITLE
allow for arbitrary tuplets / polyrhythms

### DISCUSSION THREAD

**Discussion by RegularTetragon** [SynthstromAudible/DelugeFirmware#962] [Ideas]
As it stands it's pretty easy to make polymeter stuff on the deluge but doing polyrhythms involves making unquantized live recording or a lot of manual nudging. I think more advanced tuplet support would solve this. Maybe by holding the triplet button and scrolling up or down? Tuplet nesting would be cool too but would be harder to implement

**Comment by saynotohugs** [comment]
It would be fantastic to be able to create polyrhythms on the Deluge :)
